table/block.cc

```diff
@@ -781,47 +781,45 @@ Block::Block(BlockContents&& contents, SequenceNumber _global_seqno,
     size_ = 0;  // Error marker
   } else {
     // Should only decode restart points for uncompressed blocks
-    if (compression_type() == kNoCompression) {
-      num_restarts_ = NumRestarts();
-      switch (IndexType()) {
-        case BlockBasedTableOptions::kDataBlockBinarySearch:
-          restart_offset_ = static_cast<uint32_t>(size_) -
-                            (1 + num_restarts_) * sizeof(uint32_t);
-          if (restart_offset_ > size_ - sizeof(uint32_t)) {
-            // The size is too small for NumRestarts() and therefore
-            // restart_offset_ wrapped around.
-            size_ = 0;
-          }
+    num_restarts_ = NumRestarts();
+    switch (IndexType()) {
+      case BlockBasedTableOptions::kDataBlockBinarySearch:
+        restart_offset_ = static_cast<uint32_t>(size_) -
+                          (1 + num_restarts_) * sizeof(uint32_t);
+        if (restart_offset_ > size_ - sizeof(uint32_t)) {
+          // The size is too small for NumRestarts() and therefore
+          // restart_offset_ wrapped around.
+          size_ = 0;
+        }
+        break;
+      case BlockBasedTableOptions::kDataBlockBinaryAndHash:
+        if (size_ < sizeof(uint32_t) /* block footer */ +
+                        sizeof(uint16_t) /* NUM_BUCK */) {
+          size_ = 0;
           break;
-        case BlockBasedTableOptions::kDataBlockBinaryAndHash:
-          if (size_ < sizeof(uint32_t) /* block footer */ +
-                          sizeof(uint16_t) /* NUM_BUCK */) {
-            size_ = 0;
-            break;
-          }
-
-          uint16_t map_offset;
-          data_block_hash_index_.Initialize(
-              contents.data.data(),
-              static_cast<uint16_t>(contents.data.size() -
-                                    sizeof(uint32_t)), /*chop off
-                                                   NUM_RESTARTS*/
-              &map_offset);
-
-          restart_offset_ = map_offset - num_restarts_ * sizeof(uint32_t);
-
-          if (restart_offset_ > map_offset) {
-            // map_offset is too small for NumRestarts() and
-            // therefore restart_offset_ wrapped around.
-            size_ = 0;
-            break;
-          }
+        }
+
+        uint16_t map_offset;
+        data_block_hash_index_.Initialize(
+            contents.data.data(),
+            static_cast<uint16_t>(contents.data.size() -
+                                  sizeof(uint32_t)), /*chop off
+                                                 NUM_RESTARTS*/
+            &map_offset);
+
+        restart_offset_ = map_offset - num_restarts_ * sizeof(uint32_t);
+
+        if (restart_offset_ > map_offset) {
+          // map_offset is too small for NumRestarts() and
+          // therefore restart_offset_ wrapped around.
+          size_ = 0;
           break;
-        default:
-          size_ = 0;  // Error marker
-      }
```

Comment:
These lines didn't change. The only change is that `if (compression_type() == kNoCompression)` is removed. But "make format" changed all those lines.

Reply:
But I need to do it, otherwise the format will be too bad:)